### ISSUE TITLE
App crashes on ios simulator when trying to pick image from gallery on create post page

### ISSUE THREAD

**Issue by jay11125** [PalisadoesFoundation/talawa#2368]
**Describe the bug**
In the create post page, when we try to click select the upload photo button the app crashes. I am using iPhone 15 Pro Max and iOS 17.2 for my simulator. 

**To Reproduce**

1. Go to the create post page
2. Click on the upload photo from gallery button

**Expected behavior**
The app should show gallery images so that user can select one of the images.

**Actual behavior**
The app crashes when clicking on the button.

**Screenshots**

https://github.com/PalisadoesFoundation/talawa/assets/70865848/2d581ffc-3a62-418b-991d-9c3ad83040d9

**Additional details**
Basically, the package image_picker requires permissions to be added in the info.plist file for iOS devices. It is similar to how we add permissions to AndroidManifest.xml for android devices.

**Comment by jay11125** [comment]
I would like to work on this issue @noman2002 @palisadoes @Cioppolo14
